Makefile

```diff
@@ -2,46 +2,28 @@ prefix ?= /usr/local
 bindir = $(prefix)/bin
 libdir = $(prefix)/lib
 
-REPODIR = $(shell pwd)
-BUILDDIR = $(REPODIR)/.build
+buildDir = $(shell xcodebuild -configuration Release -showBuildSettings | grep "CONFIGURATION_BUILD_DIR" | grep -oEi "\/.*")
```

Comment:
why do we need `shell` here?

Reply:
also: maybe standardize capitalization with `bindir` and `libdir`? (I like camel case, but all lower or all upper are more common with shell scripts; not sure about makefiles. Let's just be consistent, either way.)

Reply:
> why do we need `shell` here?

this will force the script to run, otherwise we'll get an empty string